.circleci/config.yml

```diff
@@ -79,3 +79,14 @@ workflows:
               ignore: /.*/
             tags:
               only: /^v.*/
+
+      # make it available to customers (only tags)
+      - architect/push-to-app-catalog:
+          name: "package and push dex-app chart"
+          app_catalog: "giantswarm-playground-catalog"
+          app_catalog_test: "giantswarm-playground-test-catalog"
+          chart: "dex-app"
+          # Trigger job on git tag.
+          filters:
+            tags:
+              only: /^v.*/
```

Comment:
I understand that this adds it to the Playground catalog. Should we have the app in two catalogs in the long term? If no, I would create an issue for team rainbow to remove it from the CP catalog.

Reply:
It is a good question. I am not 100% sure how this work. The MCs run dex via shared collection, which I am not sure if always pull from the Control Plane catalog. Something to check later. Do you want me to create the issue?

Reply:
No, I can create it. Thanks.

Reply:
Created: https://github.com/giantswarm/giantswarm/issues/19065

Also it just occured to me that as a Playground app, dex-app should be added to the repositories list https://github.com/giantswarm/docs/blob/master/scripts/aggregate-changelogs/config.yaml#L9 to appear in our aggregated changelogs.

Reply:
https://github.com/giantswarm/docs/pull/1138/files

Reply:
Nice, thanks